app/src/org/commcare/appupdate/CommcareFlexibleAppUpdateManager.java

```diff
@@ -21,7 +21,7 @@
  */
 public class CommcareFlexibleAppUpdateManager implements FlexibleAppUpdateController {
 
-    private static final String TAG = CommcareFlexibleAppUpdateManager.class.getName();
+    private static final String TAG = CommcareFlexibleAppUpdateManager.class.getSimpleName();
```

Comment:
Hey we should use more generalized tag names from `LogTypes` instead of class names as tags

Reply:
we are still writing logs with this tag.